DOCS/man/input.rst

```diff
@@ -1900,6 +1900,12 @@ Property list
 
     This property is experimental and might be removed in the future.
 
+``sub-start``
+    Return the current subtitle start time (in seconds).
+
+``sub-end``
+    Return the current subtitle end time (in seconds).
+
```

Comment:
Should document what happens if there's no current subtitle, or multiple subtitles.